### ISSUE TITLE
start: Please at least report the address to visit

### ISSUE THREAD

**Issue by yarikoptic** [vsoch/qme#26]
ATM it is just
```
$> qme start         
Database: filesystem
Server initialized for gevent.
QueueMe!
```
(well, now followed with bunch of debug output whenever I actually went there) and I had to RTFM to discover needing to go to localhost:5000 .  Please include that information in the output here.

Ideally you could even sense availability of common "starters" for the browsers, e.g. if there is a generic `xdg-open`  (should be present on the majority? of linux desktops), you could just issue `xdg-open http://localhost:5000` command and actually open the browser (yet to check what you do in helpme... ;-))

**Comment by vsoch** [comment]
It’s in the documentation, but good point, no one reads that :) will add soon.

**Comment by vsoch** [comment]
Fixed with #32
